distributed/hdfs.py

```diff
@@ -45,6 +45,24 @@ def read_block_from_hdfs(filename, offset, length, host=None, port=None,
     return bytes
 
 
+def open_file_write(paths, hdfs=None, **kwargs):
+    """ Open list of files using delayed """
+    if hdfs is None:
+        from hdfs3 import HDFileSystem
+        hdfs = HDFileSystem(kwargs.get('host'), kwargs.get('port'))
+    out = [delayed(hdfs.open)(path, 'wb') for path in paths]
+    return out
```

Comment:
I thought worth keeping around for some unknown purpose (I think the local version remains) - but of course untested and unused as it is.